nuget/WindowsAppCenter.nuspec

```diff
@@ -17,18 +17,26 @@
     <dependencies>
       <group targetFramework="uap10.0">
         <dependency id="Microsoft.NETCore.UniversalWindowsPlatform" version="6.2.8"/>
-        <dependency id="Newtonsoft.Json" version="12.0.2"/>
+        <dependency id="Newtonsoft.Json" version="13.0.1"/>
         <dependency id="SQLitePCLRaw.bundle_green" version="2.0.2" />
       </group>
       <group targetFramework="net461">
-        <dependency id="Newtonsoft.Json" version="12.0.2"/>
+        <dependency id="Newtonsoft.Json" version="13.0.1"/>
```

Comment:
Do we want to change "Newtonsoft.Json" version everywhere?

Reply:
yes